README.md

```diff
@@ -42,7 +42,7 @@ This section is about your regular work / project, not about the code in this re
 
 Think about guiding refactoring-work in your regular/project teams. Think of the day-to-day work done in your team, and improving the quality of code.
 
-You have seen a few workflows in this repository. They are quality-gates to prevent duplication, control complexity and prove functionality. Do you think some of them have potential to guide code-improvements in your team? **_enter yes/no**
+You have seen a few workflows in this repository. They are quality-gates to prevent duplication, control complexity and prove functionality. Do you think some of them have potential to guide code-improvements in your team? Yes
 
 During the next four weeks, what improvement do you plan in your code-quality practice? (in your regular work, not in this repository!)
-**_enter the practice here**
+The code in my team is kind of legacy code with very high Cyclomatic complexity, i'll try to improve the cyclomatic complexity and modularize the code so that it follows single responsibility principle.
```

Comment:
Instead of modularizing existing code, try doing this only for new extensions / additions to the code.
It's ok if the new code looks very different from the old. In fact it's a sign that it's improving :)

Reply:
Noted.